content/internships/history/contents.lr

```diff
@@ -0,0 +1,34 @@
+_model: page
+---
+_template: page-with-toc.html
+---
+title: Open Source Internships: History
+---
+body:
+
+## Google Summer of Code
+
+CC has participated in Google Summer of Code eight times before 2020. See below for more information about our projects and participation for each year.
+
+* [Google Summer of Code 2019](https://summerofcode.withgoogle.com/archive/2019/organizations/5500455663173632/)
+    * [Open Source Blog posts](http://localhost:5000/blog/categories/gsoc-2019/)
```

Comment:
This link won't work outside the locally running context.

Reply:
ugh.